packages/transformers/image/src/ImageTransformer.js

```diff
@@ -14,35 +13,46 @@ const FORMATS = new Map([
 ]);
 
 export default (new Transformer({
-  async transform({asset}) {
+  async transform({asset, options}) {
     asset.isIsolated = true;
 
-    let inputBuffer = await asset.getBuffer();
     let width = asset.query.width ? parseInt(asset.query.width, 10) : null;
     let height = asset.query.height ? parseInt(asset.query.height, 10) : null;
     let quality = asset.query.quality
       ? parseInt(asset.query.quality, 10)
       : undefined;
     let format = asset.query.as ? asset.query.as.toLowerCase().trim() : null;
 
-    let imagePipeline = sharp(inputBuffer);
-    if (width || height) {
-      imagePipeline.resize(width, height);
-    }
-
-    if (format) {
-      if (!FORMATS.has(format)) {
-        throw new Error(`Sharp does not support ${format} images.`);
+    if (width || height || quality || format) {
+      const sharp = await options.packageManager.require(
+        'sharp',
+        asset.filePath,
+        {
+          // Sharp takes too long to install for autoinstall option to make sense
+          autoinstall: false,
```

Comment:
What happens if it's not installed? Do you get a error message that explains?

Reply:
Yup, you get a resolver error with suggestions